src/services/completions.ts

```diff
@@ -179,7 +185,7 @@ namespace ts.Completions {
             return getLabelCompletionAtPosition(contextToken.parent);
         }
 
-        const completionData = getCompletionData(program, log, sourceFile, isUncheckedFile(sourceFile, compilerOptions), position, preferences, /*detailsEntryId*/ undefined, host);
+        const completionData = getCompletionData(program, log, sourceFile, isUncheckedFile(sourceFile, compilerOptions), position, preferences, /*detailsEntryId*/ undefined, host, autoImportProvider);

```

Comment:
Should getStringLiteralCompletions use autoImportProvider if present ?

Reply:
It already uses package.json data, so there’s nothing to add. I still have a long-term to-do for myself to switch it to use the PackageJsonCache, though.